x-pack/plugins/security_solution/public/common/components/event_details/threat_summary_view.tsx

```diff
@@ -5,16 +5,41 @@
  * 2.0.
  */
 
-import { EuiBasicTableColumn } from '@elastic/eui';
-import React, { useMemo } from 'react';
+import { EuiBasicTableColumn, EuiSpacer } from '@elastic/eui';
+import React from 'react';
 
-import { TimelineEventsDetailsItem } from '../../../../common/search_strategy';
-import { FormattedFieldValue } from '../../../timelines/components/timeline/body/renderers/formatted_field';
-import { BrowserFields } from '../../../../common/search_strategy/index_fields';
+import * as i18n from './translations';
 import { SummaryView } from './summary_view';
 import { getSummaryColumns, SummaryRow, ThreatSummaryRow } from './helpers';
+import { FormattedFieldValue } from '../../../timelines/components/timeline/body/renderers/formatted_field';
+import { TimelineEventsDetailsItem } from '../../../../common/search_strategy/timeline';
+import { SORTED_THREAT_SUMMARY_FIELDS } from '../../../../common/cti/constants';
 import { INDICATOR_DESTINATION_PATH } from '../../../../common/constants';
 
+const getThreatSummaryRows = (
+  data: TimelineEventsDetailsItem[],
+  timelineId: string,
+  eventId: string
+) => {
+  return data
+    .reduce((acc, { field, originalValue }) => {
```

Comment:
nit: I think if you reoriented this as `SORTED_THREAT_SUMMARY_FIELDS.map().filter()` you wouldn't need the 'insert at index' logic below.

Reply:
updated